sb/robot/markers.py

```diff
@@ -0,0 +1,62 @@
+from .game_object import GameObject
+
+import pypybox2d
+
+
+class Token(GameObject):
+    grabbable = True
+    WIDTH = 0.08
+
+    def __init__(self, arena, damping, marker_id=None):
+        self._body = arena._physics_world.create_body(position=(0, 0),
+                                                      angle=0,
```

Comment:
This indentation style, is really ugly, and not used everywhere in this project, and nowhere in the rest of our codebase. Can you change this over to the standard indenting pattern

Reply:
fixed in 0497121e0216babcca39b6eedf4d77190a7dd641